network/identity/identity.go

```diff
@@ -0,0 +1,216 @@
+package identity
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/network/event"
+	"github.com/hashicorp/go-hclog"
+	"sync"
+
+	"github.com/0xPolygon/polygon-edge/network/proto"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+const PeerID = "peerID"
+
+var (
+	ErrInvalidChainID   = errors.New("invalid chain ID")
+	ErrNotReady         = errors.New("not ready")
```

Comment:
This seems to be unused.

Reply:
It was leftover 🤦 

Fixed:

[61aa2f3](https://github.com/0xPolygon/polygon-edge/pull/343/commits/61aa2f325e24359e6f5b5b4a46d80c4c9823a95c)